src/components/UserProfile/UserProfile.jsx

```diff
@@ -1,32 +1,21 @@
 import React, {useEffect, useState} from 'react';
 import {useEasyauth} from '../../hooks/useEasyauth/useEasyauth.jsx';
+import {getProfile} from '../../api/api.js';
 
 export const UserProfile = () => {
   const auth = useEasyauth();
   const [profile, setProfile] = useState(null);
 
   useEffect(() => {
     (async () => {
-      try {
-        const token = auth.user?.access_token;
-        const response = await fetch(
-            process.env.REACT_APP_EASYAUTH_APP_URL +
-             '/tenantbackend/api/profile',
-            {
-              headers: {
-                Authorization: `Bearer ${token}`,
-              },
-            },
-        );
-        setProfile(await response.json());
-      } catch (e) {
-        console.error(e);
-      }
+      const token = auth.user?.access_token;
+      const response = await getProfile(token);
+      setProfile(response.data);
     })();
   }, [auth]);
 
   if (!profile) {
-    return <div>Loading...</div>;
+    return null;
```

Comment:
Is null ok or do we want to render something indicating an error? I am not sure what other libraries do usually.

Reply:
I have observed that, they do the same by returning null .

Reply:
Thanks